PluginHubspot/PluginHubspot.csproj

```diff
@@ -24,7 +24,6 @@
 
     <ItemGroup>
       <Folder Include="API\Replication" />
-      <Folder Include="API\Write" />
```

Comment:
Can you explain why you removed this? I'm not even entirely sure what this change does for the plugin

Reply:
I explicitly didn't remove the line, Rider (still using) did that and I didn't overwrite that change.